pkg/services/cloudmigration/cloudmigrationimpl/xorm_store_test.go

```diff
@@ -228,7 +228,7 @@ func Test_SnapshotResources(t *testing.T) {
 }
 
 func TestGetSnapshotList(t *testing.T) {
-	t.Skip("FLAKY test: disabled until fixed")
+	//t.Skip("FLAKY test: disabled until fixed")
```

Comment:
If it is fixed, can we remove this comment?

Reply:
Yep will do